source/features/index.tsx

```diff
@@ -150,6 +157,29 @@ const setupPageLoad = async (id: FeatureID, config: InternalRunConfig): Promise<
 	}
 };
 
+// Fetch hotfix releases asynchronously
+// every 6 hours ( 4 times a day )
+// this is achieved by GETting a JSON file
+// from the hotfix branch and comparing
+// with the current version of extension
+const fetchHotfixesAsynchronously = cache.function(async () => {
+	const response = await fetch('https://raw.githubusercontent.com/sindresorhus/refined-github/hotfix/hotfix.json');
+	const hotfixes = await response.json();
+
+	if (!hotfixes) {
+		return {};
+	}
+
+	if (hotfixes.unaffected) {
+		const currentVersion = browser.runtime.getManifest().version;
+		if (looseVersionCompare(hotfixes.unaffected, currentVersion) >= 0) {
+			return {};
+		}
+	}
+
+	return hotfixes;
+}, {maxAge: {hours: 6}});
```

Comment:
This needs the key I specified, or else it won't be cached as `hotfix`
```suggestion
}, {
	maxAge: {hours: 6},
	cacheKey: 'hotfix'
});
```